EMB-GUENTHER/README.md

```diff
@@ -37,11 +41,31 @@ Embedded Systems II
 - QR-Scanner für Impfzertifikate
   - QR-Code einlesen, auswerten und Aktion realisieren
 
-# buildroot
+## Vortragsthemen
+
+- Raspberry Pi Hardware
+- Buildroot / Image-Erzeugung
+- Kernelkonfiguration
+- Webserver / Motion (Stream-Software für Kamera)
+- Konfigurationsmanagement
+- WLAN<!--gemacht von Ben, -edict, Max-->
+- DHCP
+
+optional:
+
+- QR-Code, digitales Impfzertifikat, Testbestätigung
+- Servobetrieb für bewegliche Kamera
+- QEMU-Simulation des Pi 4
+
+# Buildroot
 
 [https://buildroot.org/](https://buildroot.org/)
 
+Buildroot ist ein Tool zum Bauen von Embedded Linux Images.
+
 1. [buildroot herunterladen](https://buildroot.org/downloads/buildroot-2021.02.4.tar.gz) und entpacken
-2. `make menuconfig` (`WSL2` braucht eventuell `apt install libncurses-dev`)
-3. Image bauen (`make`)
-4. Image auf SD-Karte kopieren (`sudo dd if=output/images/sdcard.img of=/dev/sdX`)
\ No newline at end of file
+2. `make <board>`, wobei \<board\> eine Konfiguration aus dem `boards`-Ordner ist (`WSL2` braucht eventuell `apt install libncurses-dev`)
```

Comment:
```suggestion
2. `make <board>`, wobei <board> eine Konfiguration aus dem `boards`-Ordner ist (`WSL2` braucht eventuell `apt install libncurses-dev`)
```

Ist das Escapen hier wirklich nötig?

Reply:
![image](https://user-images.githubusercontent.com/26359277/133141935-37ba5c68-a12b-41c5-91ce-35f992bb0bdb.png)

yes :D

Reply:
@ZeroPointMax was hast du da für ne schöne Erweiterung in VS Code?

Reply:
@Steve2955 ich glaube, die Box kommt von ``markdownlint`` von David Anson